premailer-rails.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency 'premailer', '~> 1.7', '>= 1.7.9'
-  s.add_dependency 'actionmailer', '>= 3', '< 6'
+  s.add_dependency 'actionmailer', '>= 3', '< 7'
```

Comment:
Can we merge this with the change `s.add_dependency 'actionmailer', '>= 3', '< 6.0` so that this will allow testing on Rails 6.0.0.beta2? It will help in testing before the final release.

Reply:
@prathamesh-sonpatki the current master and the current version on RubyGems has `< 6`, which allows you to test it with Rails 6.0.0.beta2. Anyway, this branch does so as well since because 6.0.0.beta2 < 6 < 7.

Reply:
Fix is needed for Rails RC1 and upcoming v6.

http://prntscr.com/nj20wh

    premailer-rails was resolved to 1.10.2, which depends on
      actionmailer (>= 3, < 6)

Reply:
For v6, yes. For release candidates, no. See above comment. From [SemVer spec item 11](https://semver.org/#spec-item-11):

> 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0

Hence, 6.0.0.beta3 < 6.0.0.rc1 < 6.0.0.

Reply:
This is strange because I'm getting this:

Bundler could not find compatible versions for gem "actionmailer":
  In Gemfile:
    ahoy_email was resolved to 1.0.3, which depends on
      actionmailer (>= 4.2)

    mailgun_rails was resolved to 0.9.0, which depends on
      actionmailer (>= 3.2.13)

    premailer-rails was resolved to 1.10.2, which depends on
      actionmailer (>= 3, < 6)

    rails (= 6.0.0.rc1) was resolved to 6.0.0.rc1, which depends on
      actionmailer (= 6.0.0.rc1)

Not sure why, from first point looks like premailer-rails.